docs/dashboard/README.md

```diff
@@ -0,0 +1,32 @@
+<!--
+title: Serverless - Dashboard Reference
+menuText: Dashboard Reference
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/dashboard/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Serverless Dashboard
+
+The [Serverless Framework Dashboard](https://dashboard.serverless.com/) is a SaaS solution that augments the Serverless Framework open source CLI to provide a powerful, unified experience to develop, deploy, test, secure and monitor your serverless applications. 
+
+The Serverless Framework free tier gives developers full access to all features included in the Serverless Framework Dashboard, but is limited to 1,000,000 function invocations per month. If you are interested in expanding your usage of the Serverless Framework beyond the free tier [contact us](https://serverless.com/enterprise/contact/) for details re available plans and pricing.
```

Comment:
Why don't we link to our pricing page here? Otherwise we need to update multiple places since this might be out of date quite soon.

Reply:
Fair point, but (1) we don't yet have a pricing page and (2) I want to make it clear that they get a meaningful amount for free.

Reply:
Makes sense. Thanks for the update :+1: